packages/react-server-website/.reactserverrc

```diff
@@ -1,19 +1,18 @@
 {
   "routes": "./routes.js",
   "host": "localhost",
-  "port": 3000,
-  "js-port": 3001,
+  "port": 3010,
```

Comment:
Oh.  Yeah, if we ever add tests for the website we'll definitely need to update the manifest. ;)